framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/DatabaseDeploymentImpl.java

```diff
@@ -136,4 +137,16 @@ public boolean ready() {
         return getInstances().size() > 0;
     }
 
+    @Override
+    public void deleteInstances(Instance... instance) {
```

Comment:
This functionality is duplicated across different deployments. Could you please move it to some common place?